src/AppHeader.tsx

```diff
@@ -32,10 +43,11 @@ export const AppHeader: React.FC = () => {
           <LanguageDropdown />
         </div>
         <div className="appheader__moltincartcontainer">
-          <span
-            className="moltin-cart-button"
-            key={id}
-          ></span>
+          <button className="epbtn --bordered" onClick={handleCartModal}>
+            {count}
```

Comment:
Let's try replicating the string message from ShopKit (Cart ()items):
![image](https://user-images.githubusercontent.com/39800563/92153333-18abc400-edf2-11ea-91e9-60015eec7ed3.png)